pkg/bootkube/create.go

```diff
@@ -48,7 +48,7 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
 
-	UserOutput("Spawning self-hosted control plane...\n")
+	UserOutput("Spawning self-hosted control plane and checkpointing components...\n")
```

Comment:
I was thinking we did since the typical k8s control plane does not include checkpointing. Reverted nonetheless.